administrator/language/de-DE/com_scheduler.ini

```
@@ -0,0 +1,141 @@
+; Joomla! German Translation
+; (C) 2005 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2022 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+COM_SCHEDULER="Geplante Aufgaben"
+COM_SCHEDULER_CONFIGURATION="Konfiguration geplanter Aufgaben"
+COM_SCHEDULER_CONFIG_FIELDSET_LAZY_SCHEDULER_DESC="Konfigurieren, wie Website-Besuche geplante Aufgaben auslösen."
+COM_SCHEDULER_CONFIG_FIELDSET_LAZY_SCHEDULER_LABEL="Lazy-Zeitplaner"
+COM_SCHEDULER_CONFIG_GENERATE_WEBCRON_KEY_DESC="Der Web-Cron benötigt einen Schlüssel zum Schutz, bevor er funktionsfähig ist. Konfiguration speichern, um den Schlüssel automatisch zu generieren."
+COM_SCHEDULER_CONFIG_GLOBAL_WEBCRON_KEY_LABEL="Globaler Schlüssel"
```

```suggestion
COM_SCHEDULER_CONFIG_GLOBAL_WEBCRON_KEY_LABEL="Sicherheitsschlüssel"
```
Ist es nicht eher ein Sicherheitsschlüssel?